config/routes/project.rb

```diff
@@ -49,6 +49,8 @@
         collection do
           get :select
           post :list
+          get :new_destroy_multiple
```

Comment:
**question:** Are these just temporary routes? Are we going to be updating the regular `destroy` route with this at some point?  Just seems strange to have routes with `new_*` prefix.

Reply:
Yeah I discussed this with Eric a while back in regards to `destroy_multiple`
I had to turn the `link_to turbo-deletes`  into `new` actions to open the modal/dialog in order to hook up the `selections` controller in a `form` so that we can have the accurate checkbox behaviour upon sample deletion. And then we decided not to create new `collection/resource routes`  because these needed a little too much from the `samples_controller` that it didn't seem worth copying over all that code. So I kept it in `samples_controller` with the `new` routes.

After realizing that the individual `destroy` also didn't work with the `checkboxes` for samples and attachments, I used similar logic. Realizing now that I likely should be able to refactor `attachments` `new destroy` into it's own collection for the individual `destroy`.